src/seedu/addressbook/parser/Parser.java

```diff
@@ -174,6 +178,44 @@ private Command prepareDelete(String args) {
         }
     }
 
+    private Command prepareUpdate(String args)  {
```

Comment:
Missing header comment. All non trivial methods should have javadoc format header comments.